addon/metrics-adapters/m-particle.js

```diff
@@ -0,0 +1,148 @@
+import objectTransforms from "../utils/object-transforms";
+import BaseAdapter from "ember-metrics/metrics-adapters/base";
+const { compact } = objectTransforms;
+import { assert } from "@ember/debug";
+import { assign } from "@ember/polyfills";
+import removeFromDOM from "ember-metrics/utils/remove-from-dom";
+
+export default BaseAdapter.extend({
+  toStringExtension() {
+    return "mParticle";
+  },
+
+  init() {
+    const config = assign({}, this.config);
+    const { apiKey, options } = config;
+    assert(
+      `[ember-metrics] You must pass a valid \`apiKey\` to the ${this.toString()} adapter`,
+      apiKey
+    );
+
+    window.mParticle = {
+      config: {
+        isDevelopmentMode: true, //switch to false (or remove) for production
```

Comment:
Will this cause issues for people using this adapter?

Reply:
good catch!  i'll get this from the config value passed from the environment.js file

Reply:
@rmi22186 just wondering if you had a bit of time to finish off this PR